client/src/pages/SearchResults/SearchResults.component.jsx

```diff
@@ -53,7 +53,12 @@ const SearchResults = () => {
             <span style={{ color: '#acb2b8', fontSize: '12px' }}>
               Results for {searchQuery}
             </span>
-            <SearchBox placeholder={'Search...'} name={'search'} pt={'mt8'} />
+            <SearchBox
+              agencyShortName={agency}
```

Comment:
nit: would prefer naming this variable `agencyShortName` so it's clearly not an object of type `Agency`